### ISSUE TITLE
How to obtain low-bitrate weights from high-bitrate weight

### ISSUE THREAD

**Issue by YuKDseele** [qingshi9974/ICLR2024-FTIC#11]
I would like to aks about how to fine-tune from a high-bitrate MSE model to a low-bitrate MSE model. 
For instance, I already have a high-bitrate model (λ=0.0483) that has been trained using the three-stage training process (2M-1M-0.2M steps) described in the paper. Now, if I want to obtain a low-bitrate version, is it sufficient to simply: 
1. Load the final trained weights
2. Adjust λ to [0.025, 0.013, 0.0067, 0.0035] 
3. Reduce the learning rate to 1e-5 
4. Perform additional training for 0.2M steps?

**Comment by qingshi9974** [comment]
> I would like to aks about how to fine-tune from a high-bitrate MSE model to a low-bitrate MSE model. For instance, I already have a high-bitrate model (λ=0.0483) that has been trained using the three-stage training process (2M-1M-0.2M steps) described in the paper. Now, if I want to obtain a low-bitrate version, is it sufficient to simply:
> 
> 1. Load the final trained weights
> 2. Adjust λ to [0.025, 0.013, 0.0067, 0.0035]
> 3. Reduce the learning rate to 1e-5
> 4. Perform additional training for 0.2M steps?

Your approach is correct. As an additional option, you could also train from scratch using different λ values, which may yield better R-D performance.

**Comment by qingshi9974** [comment]
@YuKDseele 

Based on my experience, training from scratch generally yields better performance compared to fine-tuning from the final trained weights. However, this comes at the cost of longer training time. As for the approach of beginning training midway, I haven't tried this method myself. You may consider experimenting with it in your own model and setup to see if it can achieve performance closer to that of training from scratch.

**Comment by YuKDseele** [comment]
> [@YuKDseele](https://github.com/YuKDseele)
> 
> Based on my experience, training from scratch generally yields better performance compared to fine-tuning from the final trained weights. However, this comes at the cost of longer training time. As for the approach of beginning training midway, I haven't tried this method myself. You may consider experimenting with it in your own model 和 setup to see if it can achieve performance closer to that of training from scratch.根据我的经验，与最终训练的权重相比，从头开始的培训通常会产生更好的性能。但是，这是以更长的培训时间为代价的。至于中途开始训练的方法，我自己没有尝试过这种方法。您可以考虑在自己的型号设置中进行实验，以查看它是否可以从头开始实现培训的性能。

Thank you very much for your answer. It solved my problem.
